content/klc/S4.6.adoc

```diff
@@ -1,14 +1,15 @@
 +++
-brief = "Hidden pins"
+brief = "Symbol pins must not be hidden unless they are of type 'not connected', part of a stack or the label pin of a power symbol"
```

Comment:
Below `Not Connected` is used with regards to pin type. Should that be consistent? It feels like it to me.

Reply:
done

Reply:
I could have been more clear that there were two parts I was talking about being missing here but used below. Both capitalization and the code format (enclosed in grace accent marks) is used below. Capitalization is done but code style is still missing.

Reply:
I am not sure if the brief part can use utf8 chars. My guess is that the normal quote mark is used on purpose here. Maybe @nickoe or @marekr know more about that. (I am not even sure where the brief part is shown to be honest.)

Reply:
I see. It appears to be the section header.

What about retaining the section header `Hidden pins` and then moving the sentence down into this rule? As it is, that appears to be an awfully long name to section S4.6.

Reply:
Ah you are right this is the header. (It is so long ago that i started this that i already forgot half of what i intended.)

I assume i had a good reason why i changed it. Is it possible that this wording matches travis output? I will check that tomorrow as it is way too late for me for today.

Reply:
Perhaps. I wonder if such a verbose section header is a good idea, though? We already have the rule number to cross-reference with check script output.

Reply:
i reverted the title as i am currently not finding something better that fulfils your requirement for a short one.